presto-main/src/test/java/com/facebook/presto/sql/planner/TestQuantifiedComparison.java

```diff
@@ -96,8 +96,7 @@ private void assertQuantifiedComparison(String query)
         assertPlan(query, anyTree(
                 node(JoinNode.class,
                         tableScan("orders"),
-                        anyTree(
```

Comment:
I think one should finally fix this, to make `anyTree` accept empty tree. Would you accept this honor?